pinot-core/src/main/java/org/apache/pinot/core/minion/rollup/MergeRollupSegmentConverter.java

```diff
@@ -177,7 +177,7 @@ public Builder setSegmentName(String segmentName) {
     }
 
     public MergeRollupSegmentConverter build() {
-      return new MergeRollupSegmentConverter(_inputIndexDirs, _workingDir, _tableName, _segmentName, _mergeType,
+      return new MergeRollupSegmentConverter(_inputIndexDirs, _workingDir, _tableName, _segmentName, _collectorType,
```

Comment:
`MergeRollupSegmentConverter` is actually the old code and we should deprecate this. We now need to use `SegmentProcessorFramework`. Can you add the comment? We can address this from another PR. Please refer to `RealtimeToOfflineSegmentsTaskExecutor` on how to use segment processing framework.

Reply:
Added the comment for this class.